articles/cross_compilation_build_tools_design.md

```diff
@@ -0,0 +1,268 @@
+---
+layout: default
+title: Cross-compiling ROS 2 packages
+abstract:
+  This article is a design proposal for extending the build tools for ROS 2 to
+  simplify cross-compiling ROS 2 packages for platforms currently not supported
+  by ROS 2.
+author: >
+  [Thomas Moulard](https://github.com/thomas-moulard),
+  [Juan Rodriguez Hortala](https://github.com/juanrh)
+published: false
+---
+
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+## Background
+
+The [cross-compilation tutorial](https://index.ros.org/doc/ros2/Tutorials/Cross-compilation)
+and the [steve/ros2_raspbian_tools repository](https://github.com/esteve/ros2_raspbian_tools)
+contain instructions for cross-compiling ROS 2 to unsupported platforms like
+ARM64 and ARM-HF, using some configuration files for CMake and Docker.
+Other projects like [steve/ros2_raspbian_tools](https://github.com/esteve/ros2_raspbian_tools),
```

Comment:
Looks like
`[steve/ros2_raspbian_tools](https://github.com/esteve/ros2_raspbian_tools)` is already mentioned above. Did you mean https://github.com/esteve/ros2_objc  (for iOS) ?